server/app.js

```diff
@@ -48,7 +48,7 @@ app.use(passport.session());
 passport.use(
 	new GitHubStrategy(
 		{
-			clientID: process.env.SCOREBOARD_GH_APPID,
+			clientID: process.env.SCOREBOARD_GH_APPID || "REMOVE_ME",
```

Comment:
Why is this called "REMOVE_ME"?

Reply:
@maniarathi  This is just a hack to get things running locally. I think, there is an issue getting the clientID when running locally, but I havent dug into the details. 
I dont think we should keep the workaround when running in prod, do you think it would be better to add a todo?

Reply:
Yeah you could open up a new ticket so that we remember it's there. Thanks!